app/views/declaration/seal_remove.scala.html

```diff
@@ -17,56 +17,92 @@
+@this(
+        govukLayout: gdsMainTemplate,
+        sectionHeader: sectionHeader,
+        govukFieldset: GovukFieldset,
+        pageTitle: pageTitle,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton,
+        govukTable: GovukTable,
+        errorSummary: errorSummary,
+        saveButtons: saveButtons,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[YesNoAnswer], containerId: String, sealId: String)(implicit request: Request[_], messages: Messages)
 
+@sealsTable = {
 
-@main_template(
-    title = Title(messages("standard.seal.remove.title", containerId))
-) {
-
-    @components.back_link(controllers.declaration.routes.SealController.displaySealSummary(mode, containerId), mode)
-
-    @components.error_summary(form.errors)
+    @govukTable(Table(
+        rows = Seq(
+            Seq(
+                TableRow(
+                    content = Text(sealId)
+                )
+            )
+        ),
+        head = Some(List(
+            HeadCell(
+                content = Text(messages("declaration.seal.summary.heading")),
+                colspan = Some(2)
+            )
+        )),
+        attributes = Map("id" -> "seal-table")
+    ))
+}
 
-    @components.heading(title = messages("standard.seal.remove.title", containerId))
+@removeSealRadios = {
+        @govukRadios(Radios(
+        name = "yesNo",
+        items = Seq(
+            RadioItem(
+                id = Some("Yes"),
+                value = Some(yes),
+                content = Text(messages("site.yes")),
+                checked = form("yesNo").value.contains(yes)
+            ),
+            RadioItem(
+                id = Some("No"),
+                value = Some(no),
+                content = Text(messages("site.no")),
+                checked = form("yesNo").value.contains(no)
+            )
+        ),
+        errorMessage = form("yesNo").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args: _*))))
+    ))
+}
 
-    @helper.form(SealController.submitSealRemove(mode, containerId, sealId), 'autoComplete -> "off") {
-        @helper.CSRF.formField
+@govukLayout(
+    title = Title("declaration.seal.remove.title", "", containerId),
+    backButton = Some(BackButton(messages("site.back"), controllers.declaration.routes.SealController.displaySealSummary(mode, containerId)))) {
 
-        <div class="field-group mb-2">
-            <table id="seal-table">
-                <thead>
-                    <tr>
-                        <th>@messages("standard.seal.remove.heading")</th>
-                    </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <td >@sealId</td>
-                </tr>
-                </tbody>
-            </table>
-        </div>
+    @formHelper(action = SealController.submitSealRemove(mode, containerId, sealId), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
```

Comment:
Don't we need  `@helper.CSRF.formField` here?

Reply:
FormHelper contains the CSRF token inside